### DISCUSSION TITLE
It seems unintuitive to remove archives.files defaults

### DISCUSSION THREAD

**Discussion by fasmide** [goreleaser/goreleaser#4311] [Ideas, Issue Triage, and General discussions]
Hello

I'm releasing a build that contains a go package called `license/` - by go releasers default, this directory will be included in the builds - this seems like a sane default; however - it's really not that intuitive to remove these defaults.

First, I tried to set `files:` variable to an empty array.
```
archives:
  - id: default
    files: []

```
But I think the way go releaser works is it checks for array size, and if it's empty - inserts its own defaults - knowing this - I can ofcause just specify some junk, like so

```
archives:
  - id: default
    files: 
      - blarh*

```
And all is good - as long as I don't introduce any "blarh" files to the repo - but there must be a better way :)

Thanks for a great product

**Comment by caarlos0** [comment]
yes, unfortunately this is ancient behavior and would require a breaking change to fix.

will likely happen in v2.
